src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,96 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+
+    private static final int INITIAL_CAPACITY = 10;
+    private Object[] elementData = new Object[INITIAL_CAPACITY];
```

Comment:
```suggestion
    private T[] elementData;
```
use type parameter, initialize in constructor